modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/source/SourceExtensionObjectFactory.java

```diff
@@ -44,7 +44,7 @@
  *
  * @since 4.0
  */
-public class ExtensionSourceObjectFactory extends AbstractExtensionObjectFactory<ExtensionMessageSource> {
+public class SourceExtensionObjectFactory extends AbstractExtensionObjectFactory<ExtensionMessageSource> {
```

Comment:
Bad rename. The term `Extension` is qualifying the `Source` not the `Source` qualifying the `Extension`